content/knowledge-white-label/white-label-scripts.md

```diff
@@ -145,7 +145,12 @@ curl -XPOST -H 'x-auth-token: <your-auth-token>' \
        'https://api.codemagic.io/apps/<app-id>/variables'
 {{< /highlight >}}
 
+{{<notebox>}}
+**Note:** Use this command if you want to add a base64 encoded value. 
```

Comment:
This might cause confusion. Which values should or shouldn't be base64 encoded? I would mention the .keystore. In the case of a .env file, I would say that it's encoded to preserve the file formatting.